docs/actions.md

```diff
@@ -216,10 +216,11 @@ ok: invoked /whisk.system/samples/greeting with id 5975c24de0114ef2b5c24de0118ef
 A blocking invocation request will _wait_ for the activation result to be available. The wait period
 is the lesser of 60 seconds or the action's configured
 [time limit](reference.md#per-action-timeout-ms-default-60s).
-The result of the activation is returned if it is available within the wait period.
-Otherwise, the activation continues processing in the system and an activation ID is returned
-so that one may check for the result later, as with non-blocking requests
-(see [here](#watching-action-output) for tips on monitoring activations).
+The result of the action is successfully returned if it is available within the wait period.
+Otherwise, the result is an error caused by the action timeout and the response status is _application error_.
```

Comment:
This is not quite true. The API response is 202 (accepted) and an activation id, not an application error (which would imply the activation completed) if the blocking period (60s) expires before a result is ready.

Reply:
This is about blocking invocation. Here's my test:
`wsk action create sleep tests/dat/actions/sleep.js -t 100`
`wsk action invoke sleep --param sleepTimeInMs 1100 --blocking`
The response code is 502 and response itself is:
`ok: invoked /guest/sleep with id 75043a3d2eec4556843a3d2eec855620
{
    "activationId": "75043a3d2eec4556843a3d2eec855620",
    "annotations": [
        {
            "key": "path",
            "value": "guest/sleep"
        },
        {
            "key": "waitTime",
            "value": 1014
        },
        {
            "key": "kind",
            "value": "nodejs:6"
        },
        {
            "key": "limits",
            "value": {
                "logs": 10,
                "memory": 256,
                "timeout": 100
            }
        },
        {
            "key": "initTime",
            "value": 152
        }
    ],
    "duration": 255,
    "end": 1533718832936,
    "logs": [],
    "name": "sleep",
    "namespace": "guest",
    "publish": false,
    "response": {
        "result": {
            "error": "The action exceeded its time limits of 100 milliseconds."
        },
        "status": "application error",
        "success": false
    },
    "start": 1533718832681,
    "subject": "guest",
    "version": "0.0.1"
}
`

Reply:
Ah, the text reads as if you're referring to the blocking wait period. We wait for 60s at most for a blocking invocation to finish, after that we fall back to a 202 (actions can be configured to run longer than 60s). That's what @rabbah is referring to.

Reply:
I see. Let me rephrase it then.